bundle/internal/tf/codegen/.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Regenerate schema",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "main.go",
+            "cwd": "."
+        },
+    ],
+}
```

Comment:
Fair point, I included this in because:
1. We already do this for terraform: https://github.com/databricks/terraform-provider-databricks/blob/master/.vscode/launch.json
2. Unlikely developers would have a custom setting in this directory for regenerating the structs based on the new provider version. 

Just a small QOL improvement, I am fine with taking it out.

Reply:
Yeah let's just add it in .gitignore instead